extensions/amp-access/0.1/test/test-amp-access.js

```diff
@@ -1493,7 +1470,9 @@ describes.fakeWin('AccessService multiple sources', {
     document = win.document;
     clock = sandbox.useFakeTimers();
     clock.tick(0);
-
+    // Polyfill performance.now() method as it is not available in fakeWin
+    // environment.
+    win.performance = {now: () => Date.now()};
```

Comment:
You could also just add it to `fake-dom.js#FakeWindow`.

Reply:
done. thanks!

Reply:
Looks like there are still a bunch of these in this file.

Reply:
right..my editor cache needed to be cleared..for some reason it was showing my updated changes but this file wasn't actually edited. Fixed.